### PR TITLE
Add installation steps to Docs

### LINKED ISSUE #1: Add installation steps to Docs
Describe how to install this package using SPM and Cocoapods

### PR DESCRIPTION
**Pull request by mariorsjr**
Added the installation steps to Readme file
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+import PaintAndBrush
+```
+### Swift Package Manager(Xcode 11+)
+The Swift Package Manager is a tool for automating and managing the distribution of 3rd-party Swift code. Starting on Xcode 11, SPM is natively integrated with Xcode.
+
+To integrate PaintAndBrush into your project using SPM open your project using Xcode 11+. Then click on `File` -> `Swift Packages` -> `Add Package Dependency` and enter the PaintAndBrush repository's URL (https://github.com/mugbug/PaintAndBrush).
+
+To use PaintAndBrush into your project, import PaintAndBrush with:
+```swift
+import ViewCodeHelper
```
Comment (mugbug) [inline README.md:29]: I changed the project name but the package is still using the previous one? 😔

**Review by mugbug — APPROVED** [review]
Perfect! Thank you so much for your contribution!

File: README.md (hunk 1 above)
Comment (mariorsjr) [reply README.md:29]: Yes, still is using the old name :/ 
I actually saw the name change and give a try on `import PaintAndBrush` but Xcode give an error. 
Only worked with ViewCodeHelper.

Comment (mugbug) [reply README.md:29]: Hm, hey @mariorsjr, I think I was able to figure that out. The thing is I changed the project name but forgot to create a version for that 😅 . Can you give it another try with v0.2.0, please?

Comment (mariorsjr) [reply README.md:29]: @mugbug actually looks like it was my mistake. I checked out to version 0.1.4. 
Now I tried to give a checkout to version 0.2.0 and worked with `import PaintAndBrush`
I will update the readme.

Comment (mugbug) [reply README.md:29]: I just pushed this new version, that's why you didn't get that in the first place. Thanks!

**Pull request merged**